.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  "extends": "airbnb",
+  "plugins": [
```

Comment:
oh yes that's right @p0wl. Use the eslint command it sort of added it by default. Fixed it now